pkg/note/note_test.go

```diff
@@ -79,24 +80,28 @@ fycQ0e/nw1JkypGoL1j4cwM=
 func TestKeyHash(t *testing.T) {
 	origin := "testkey"
 	tests := []struct {
-		name          string
-		key           []byte
-		expectedKeyID uint32
+		name             string
+		key              []byte
+		expectedKeyID    uint32
+		expectedLogIDHex string
```

Comment:
Since LogID is meant to be bytes, is it still supposed to be hex-encoded first? It will be encoded again to base64 during transmission.

 - https://github.com/sigstore/protobuf-specs/blob/a4c70fea6889694dfbceea11e0f9147335a95d79/protos/sigstore_common.proto#L144-L145

Reply:
This is just for tests to store the log ID as a string. The log ID is bytes and JSON marshalled to a base64-encoded string in the response.

Reply:
Ok, It would be better to use bas64 in these examples.

Reply:
I've changed this to call `hexDecodeOrDie` rather than pass around the hex-encoded ID to avoid confusion. I'm leaving this as a hex string since we aren't testing anything with the bundle and this is typical for digest output.

Reply:
Ok. I'm not blocking, but there's also `b64DecodeOrDie`.